config/projects-prod/pegs-challenge.yaml

```diff
@@ -0,0 +1,53 @@
+stack_name: pegs-challenge
+
+stack_tags:
+  Department: DNT
+  Project: PEGS Challenge
+  OwnerEmail: nextflow-admins@sagebase.org
+  CostCenter: 12470001 # Valid values here: https://github.com/Sage-Bionetworks/aws-infra/tree/master/templates/tags
```

Comment:
In some of the other examples, I see tags formatted like `cost-center-name / 123456` - is the cost center name necessary or can I leave it like this?

Reply:
Good catch, this needs to be modified. I posted in #sageit

Reply:
@jaymedina : "PEGS Gene Environment / 124700"